whylog_vim/output_formater/utils.py

```diff
@@ -0,0 +1,7 @@
+def convert_to_buttons(elem_list):
+    if len(elem_list) > 0:
```

Comment:
I suggest to refactor to avoid long block of code inside if.
```
if len(elem_list) == 0:
    return ''
result = '[%s]' % elem_list[0]
for elem in elem_list[1:]:
    result += '\n[%s]' % elem
return result
```